webapp/src/main/webapp/themes/wilma/css/wilma.css

```diff
@@ -1733,3 +2064,384 @@ span.subclassExpandMinus {
+    }
+
+    #helptext {
+        width: 70%;
+        padding-right: 10px;
+    }
+    
+}
+
+#cutofflabel {
+    width: initial;
+}
+
+#queryform input[type="text"], select, input[type="password"]{
+    max-width: 100%;
+}
+
+#queryCutoff {
+    margin-left: -54px;
+}
+
+/* Author Network */
+
+.propertyTabsList {
+    display: flex;
+}
+
+#body {
+    min-width: initial;
+}
+
+#chord {
+    width: 100%;
+    height: 100%;
+    float: left;
+    overflow: auto;
+}
+
+
+#chord > svg {
+    transform-origin: 0 50%;
+    -webkit-transform-origin: 0 50%;
+    -ms-transform-origin: 0 50%;
+    transform-origin: 0 50%;
+    height: 100%;
+}
+
+@media (max-width: 360px) {
+    #body > #bodyPannel {
+        height: 360px;
+    }
+
+    #chord > svg > * {
+        scale: 0.3;
+        overflow: hidden;
+    }
+    
+    #chord-info-div {
+        scale: 0.7;
+    }
+}
+
+@media (min-width: 361px) and (max-width: 399px) {
+    #body > #bodyPannel {
+        height: 360px;
+    }
+
+    #chord > svg > * {
+        scale: 0.4;
+    }
+    
+    #chord-info-div {
+        scale: 0.7;
+    }
+}
+
+
+@media (min-width: 400px) and (max-width: 449px) {
+    #body > #bodyPannel {
+        height: 400px;
+    }
+    
+    #chord > svg > * {
+        scale: 0.44;
+    }
+    
+    #chord-info-div {
+        scale: 0.7;
+    }
+}
+
+
+
+@media (min-width: 450px) and (max-width: 499px) {
+    #body > #bodyPannel {
+        height: 450px;
+    }
+    
+    #chord > svg > * {
+        scale: 0.5;
+    }
+    
+    #chord-info-div {
+        scale: 0.7;
+    }
+}
+
+
+
+@media (min-width: 500px) and (max-width: 575px) {
+    #body > #bodyPannel {
+        height: 500px;
+    }
+    
+    #chord > svg > * {
+        scale: 0.55;
+    }
+    
+    #chord-info-div {
+        scale: 0.8;
+    }
+}
+
+@media (min-width: 576px) and (max-width: 767px) {
+    #body > #bodyPannel {
+        height: 576px;
+    }
+    
+    #chord > svg > * {
+        scale: 0.6;
+    }
+}
+
+
+@media (min-width: 768px) and (max-width: 991px) {
+    #body > #bodyPannel {
+        height: 768px;
+    }
+    
+    #chord > svg > * {
+        scale: 0.8;
+    }
+}
+
+
+
+#sparkline-container-full > #publication-count-sparkline-include {
+    width: 100%;
+    margin-bottom: 0px;
+}
+
+
+#sparkline-container-full > #coauthor-count-sparkline-include {
+    width: 100%;
+    margin-bottom: 0px;
+}
+
+#sparkline-container-full .google-visualization-sparkline-default img.google-visualization-sparkline-image {
+    max-width: 100%;
+}
+
+
+#body .vis-tables {
+    width: 100%;
+    margin-bottom: 0px;
+}
+
+@media (min-width: 768px) {
+    #sparkline-container-full > #publication-count-sparkline-include {
+        width: 50%;
+        margin-bottom: 15px;
+    }
+    #sparkline-container-full > #coauthor-count-sparkline-include {
+        width: 50%;
+        margin-bottom: 15px;
+    }
+    #body .vis-tables {
+        width: 48%;
+        margin-bottom: 15px;
+    }
+}
+
+#graphContainer {
```

Comment:
I suspect this is the problem. If this #graphContainer css is removed then the temporal graph works again.